### PR TITLE
Order, User, PersonalInfo classes added

### PR DESCRIPTION
**Pull request by vbre**

### CODE REVIEW

File: Solution/ContosoProject/Domain/Entities/Order.cs [hunk 1]
```diff
@@ -6,7 +6,34 @@
...
-    public class Order
+    public class Order : BaseEntity
     {
+        public Customer customer;
+        public List<GoodsRow> goodsList = new List<GoodsRow>();
+        public double totalCost;
+        public OrderStatus currentStatus;
+        public List<Comment> comments;
+
+        public Order(Customer currentCustomer, List<GoodsRow> good) : this (currentCustomer, good, new Comment())
```
Comment (vbre) [inline Solution/ContosoProject/Domain/Entities/Order.cs]: maybe this isn't needed

File: Solution/ContosoProject/Domain/Entities/Order.cs [hunk 2]
```diff
@@ -6,7 +6,34 @@
...
 {
-    public class Order
+    public class Order : BaseEntity
     {
+        public Customer customer;
+        public List<GoodsRow> goodsList = new List<GoodsRow>();
+        public double totalCost;
+        public OrderStatus currentStatus;
+        public List<Comment> comments;
+
```
Comment (vbre) [inline Solution/ContosoProject/Domain/Entities/Order.cs]: maybe such ctor is not needed

File: Solution/ContosoProject/Domain/Entities/Order.cs [hunk 3]
```diff
@@ -6,7 +6,34 @@
 
 namespace Domain.Entities
 {
-    public class Order
+    public class Order : BaseEntity
     {
+        public Customer customer;
```
Comment (demmy) [inline Solution/ContosoProject/Domain/Entities/Order.cs]: public property name must be Capitalize (Pascal case) - coding standart
please dont use auto property, use simple public property instead
example: **public OrderStatus OrderStatus {get; set;}**

**everywhere!**

File: Solution/ContosoProject/Domain/Entities/Order.cs [hunk 4]
```diff
@@ -6,7 +6,34 @@
...
+        public Order(Customer currentCustomer, List<GoodsRow> good) : this (currentCustomer, good, new Comment())
+        {
+        }
+
+        public Order(Customer customer, List<GoodsRow> goods, Comment comment)
+        {
+            this.customer = customer;
+            if (goods != null)
+            {
+                foreach (GoodsRow row in goods)
```
Comment (demmy) [inline Solution/ContosoProject/Domain/Entities/Order.cs]: move this to private method

File: Solution/ContosoProject/Domain/Entities/PersonalInfo.cs [hunk 5]
```diff
@@ -6,7 +6,53 @@
...
+    public class PersonalInfo : BaseEntity
     {
+        private string firstName;
+        private string middleName;
+        private string lastName;
+
+        public string FirstName
+        {
+            get { return firstName; }
+            set
```
Comment (demmy) [inline Solution/ContosoProject/Domain/Entities/PersonalInfo.cs]: validation logic on POCO model are redundant

File: Solution/ContosoProject/Domain/Entities/User.cs [hunk 6]
```diff
@@ -6,7 +6,47 @@
...
+        private string login;
+        private string password;
+        public Role role;
+
+        public string Login
+        {
+            get { return login; }
+            set
+            {
+                if (value.Length != 0 && value.Length >= 4)
```
Comment (demmy) [inline Solution/ContosoProject/Domain/Entities/User.cs]: why 4 is magic number ;)

**Comment by demmy** [comment]
be disscussed on lecture

File: Solution/ContosoProject/Domain/Entities/User.cs (hunk 6 above)
Comment (vbre) [inline Solution/ContosoProject/Domain/Entities/User.cs]: I forgot to delete this part))) it was added just for joke))

File: Solution/ContosoProject/Domain/Entities/Order.cs (hunk 3 above)
Comment (vbre) [inline Solution/ContosoProject/Domain/Entities/Order.cs]: I don't think that all variables must be properties, so I named them as just an ordinary variables, without capitalization

**Comment by vbre** [comment]
All issues are fixed

**Comment by demmy** [comment]
@vbre please merge conflicts

**Comment by vbre** [comment]
I'm sorry what conflicts, everything was merged without any problems yesterday...

**Pull request merged**